core/internal/features_test.go

```diff
@@ -863,8 +863,8 @@ func TestIntegration_FluxMonitor_Deviation(t *testing.T) {
 
 	// Configure fake Eth Node to return 10,000 cents when FM initiates price.
 	eth.Context("Flux Monitor initializes price", func(mock *cltest.EthMock) {
-		mock.Register("eth_call", "10000") // 10,000 cents
-		mock.Register("eth_call", "0x1")   // aggregator round: 1
+		mock.Register("eth_call", cltest.MustEVMUintHexFromBase10String(t, "10000")) // 10,000 cents
```

Comment:
**Review note:** it doesn't seem that Geth ever encodes `eth_call` responses in any format other than ABI encoding, which means that these mocked values are incorrect.  Instead, we convert them to left-padded hex uints.